content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: The Same Subject Continued: Concerning Dangers from Foreign Force and Influence
+date: "2020-11-18"
+originalAuthor: John Jay
+description: "It is better to have a single, united government when it comes to the dangers of war an international relations."
+---
+
+It hardly takes a genius to recognize that the citizens of any country (especially intelligent and well-informed citizens like Americans) rarely adopt incorrect opinions about what is good for them--at least not for very long. So, the fact that our countrymen have long been united in believing that we should stay united under a single, sufficiently powerful federal government means that it's probably a good idea.
```

Comment:
For the sake of reviewing and ease of editing, I have been splitting up my markdown paragraphs on multiple lines. If you do any more of these, you might consider it.

A few comments on this line:

1. the parenthetical seems a bit sarcastic. Is that intentional?
2. Use `&mdash;` instead of `--`
3. The last sentence still feels a bit archaic, especially "countrymen" and "have long been united in believing".